config/initializers/content_security_policy.rb

```diff
@@ -1,26 +1,25 @@
 # Be sure to restart your server when you modify this file.
 
-# Define an application-wide content security policy
-# For further information see the following documentation
-# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+# Define an application-wide content security policy.
+# See the Securing Rails Applications Guide for more information:
+# https://guides.rubyonrails.org/security.html#content-security-policy-header
 
-Rails.application.configure do
```

Comment:
We'll want to reinstate this configuration

Reply:
:+1:  Done. The only changes are the comments.